models/repo_archiver.go

```diff
@@ -57,7 +57,7 @@ func (archiver *RepoArchiver) RelativePath() (string, error) {
 		return "", err
 	}
 
-	return fmt.Sprintf("%s/%s/%s.%s", repo.FullName(), archiver.CommitID[:2], archiver.CommitID, archiver.Type.String()), nil
+	return fmt.Sprintf("%d/%s/%s.%s", repo.ID, archiver.CommitID[:2], archiver.CommitID, archiver.Type.String()), nil
```

Comment:
Why this change?

You don't need to load the repo anymore and can just use the archiver.RepoID unless you really want to check if the repo is still extant.